Code/GraphMol/FMCS/FMCS.cpp

```diff
@@ -111,6 +111,8 @@ void parseMCSParametersJSON(const char* json, MCSParameters* params) {
         "RingMatchesRingOnly", p.AtomCompareParameters.RingMatchesRingOnly);
     p.BondCompareParameters.RingMatchesRingOnly = pt.get<bool>(
         "RingMatchesRingOnly", p.BondCompareParameters.RingMatchesRingOnly);
+    p.AtomCompareParameters.CompleteRingsOnly = pt.get<bool>(
```

Comment:
This is a little bit misleading since `CompleteRingsOnly` will apply to both to both atoms and bonds.
Would it maybe be clearer to call this one `CompleteAtomRingsOnly`?
(To be clear: I'm not sure... asking for your opinion)

Reply:
Right, I hadn't looked at that code carefully enough and I thought bond and atom parameters came from different subtrees, but that's not the case.
I'd prefer `AtomCompleteRingsOnly` if you agree.
However, now I realize that at the moment it is also not possible to set `RingMatchesRingOnly` separately on bonds and atoms from JSON, whereas it is possible from C++ and Python.
I suggest the following:
* `RingMatchesRingOnly` sets both `AtomCompareParameters.RingMatchesRingOnly` and `BondCompareParameters.RingMatchesRingOnly`, in analogy to the `ringMatchesRingOnly` parameter in one of `findMCS` overloads
* `BondRingMatchesRingOnly` sets only `BondCompareParameters.RingMatchesRingOnly`
* `AtomRingMatchesRingOnly` sets only `AtomCompareParameters.RingMatchesRingOnly`
* `CompleteRingsOnly` sets `BondCompareParameters.CompleteRingsOnly` but not `AtomCompareParameters.CompleteRingsOnly`, in analogy to the `completeRingsOnly` parameter in one of `findMCS` overloads
* `BondCompleteRingsOnly` sets only `BondCompareParameters.CompleteRingsOnly`
* `AtomCompleteRingsOnly` sets only `AtomCompareParameters.CompleteRingsOnly`

Does that sound reasonable to you?

Reply:
yep, sounds good

Reply:
Done, and updated JSON tests.